metaci/testresults/importer.py

```diff
@@ -56,20 +60,18 @@ def import_test_results(build_flow, results):
 
         testclass = classes.get(result['ClassName'], None)
         if not testclass:
-            try:
-                testclass = TestClass.objects.get(name = result['ClassName'], repo=build_flow.build.repo)
-            except TestClass.DoesNotExist:
-                testclass = TestClass(name = result['ClassName'], repo=build_flow.build.repo)
-                testclass.save()
+            testclass, created = TestClass.objects.get_or_create(
+                name = result['suite']['name'],
```

Comment:
I don't think this exists for non-robot tests imported using load_junit

Reply:
I added a new JUnit test_type choice and changed import_test_results to require a test_type value.  Tests loaded via JUnit will get the test_type JUnit now.

Reply:
My point is that `load_junit` returns a dict that does not contains a `suite` key.

Reply:
Ah, I see.  Fixed in latest commit.